Lesson4.1/src/Computer.java

```diff
@@ -0,0 +1,75 @@
+import java.util.Random;
+
+public class Computer {
+    private final String CPU = "AMD 4GHz 8 core";
+    private final String SSD = "512GB";
+    private final String RAM = "8GB";
+    private int countLifeCycle = 10;
+    private int randomNumber;
+    private ComputerCondition computerCondition;
+    Random random = new Random();
+
+    public Computer() {
+        randomNumber = random.nextInt(2);
+    }
+
+    {
+        computerCondition = ComputerCondition.STATE_OFF;
+    }
+
+    public void turnOn(Integer integer) {
```

Comment:
you should read int from the console here, in the computer. But only if it's really needed (for STATE_OFF here and for STATE_ON in turnOff)